### DISCUSSION TITLE
export `loaderArgs` and `actionArgs` types

### DISCUSSION THREAD

**Discussion by SteveSuv** [remix-run/remix#9530] [Proposals]
when I use `defineLoader` and `defineAction`, I want to use `args` types in other functions like `handleRequest` out of `loader`, but I can't find the type export.

![image](https://github.com/remix-run/remix/assets/35391006/18e7b706-0709-4cee-b667-455204eb40c8)

**Comment by binaryartifex** [comment]
im waiting on these to be exported as well as the Serializable type. for the interim though im just doing the following to extract those types out of their respective function types

```typescript
import {
  unstable_defineAction as defineAction,
  unstable_defineLoader as defineLoader,
} from "@remix-run/node";

type Loader = Parameters<typeof defineLoader>[0];
type Action = Parameters<typeof defineAction>[0];

export type LoaderArgs = Parameters<Loader>[0];
export type ActionArgs = Parameters<Action>[0];
```
